client/components/jetpack/daily-backup-status/status-card/backup-just-completed.tsx

```diff
@@ -45,9 +44,9 @@ const BackupJustCompleted: React.FC< Props > = ( { justCompletedBackupDate, last
 			</p>
 
 			{ siteLastBackupDate && (
-				<div className="status-card__no-backup-last-backup">
+				<p className="status-card__no-backup-last-backup">
 					{ translate( 'Last backup before today: {{link}}%(lastBackupDisplayDate)s{{/link}}', {
-						args: { lastBackupDisplayDate: preventWidows( lastBackupDisplayDate ) },
```

Comment:
Same as above, but check message once backup is complete:

<img width="318" alt="Screenshot 2025-05-08 at 3 07 05 pm" src="https://github.com/user-attachments/assets/6c2f1657-a11b-41be-b596-daeafdd14fc2" />